pkg/bench/rttanalysis/validate_benchmark_data.go

```diff
@@ -319,7 +319,9 @@ func (b benchmarkExpectations) find(name string) (benchmarkExpectation, bool) {
 }
 
 func (e benchmarkExpectation) matches(roundTrips int) bool {
-	return e.min <= roundTrips && roundTrips <= e.max
+	return (e.min <= roundTrips && roundTrips <= e.max) ||
+		e.min == e.max &&
```

Comment:
would
```
(e.min <= roundTrips && roundTrips <= e.max) || (e.min == e.max && (roundTrips == e.min-1 || roundTrips == e.min+1))
```

be even more clear?

Reply:
I unwrapped it one level and added commentary